NGitLab/IEpicClient.cs

```diff
@@ -21,6 +21,13 @@ public interface IEpicClient
         /// <returns></returns>
         Epic Get(int groupId, int epicId);
 
+        /// <summary>
+        /// Return all issues that are assigned to an epic
+        /// </summary>
+        /// <param name="groupId">Group ID</param>
+        /// <param name="epicId">Epic ID</param>
+        IEnumerable<Issue> GetIssues(int groupId, int epicId);
```

Comment:
Now that we have async implementation, you need to use it :)

```suggestion
        GitLabCollectionResponse<Issue> GetIssuesAsync(int groupId, int epicId);
```

Reply:
comme aucun autres des methodes sont async, ca ne semble pas trop uniforme de n'en mettre que une.  Ca va rend sont utilisation weird, non?

Reply:
Nope, it's the new way of doing it! Don't get fooled by the async suffix, `GitLabCollectionResponse` implements `IEnumerable` and `IAsyncEnumerable`, so the consumer can choose how they want to enumerate the result.

For the details, we had no choice to suffix the method name because we could not change existing methods. So, while the method ends with "Async", it is not actually async. It is async when you use `await foreach`.